### ISSUE TITLE
추출한 csv 파일 관련 글자 문제

### ISSUE THREAD

**Issue by Jsueeee** [euphony-io/check-in-listener#33]
![image](https://user-images.githubusercontent.com/66951780/135717637-f4071229-d593-4802-9a34-93ea0df643e5.png)

@sujin9 수진님 혹시 추출한 파일에서 이런 거 발견하셨나요??

**Comment by sujin9** [comment]
아니요 처음 봅니다..! 아니면 혹시 CSV 파일 열 때 인코딩 방식 다르게 선택 가능하신가요? UTF-8이나 ASCII 로요!
우선 저도 다시 해보겠습니다!

**Comment by Jsueeee** [comment]
혹시 단말에서 무슨 앱으로 파일 열으셨나요??

**Comment by sujin9** [comment]
저는 한컴오피스 viewer 앱 사용했는데,
구글 스프레드시트앱도 다운받아서 한 번 해보려구요 !
